app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -62,6 +64,12 @@ open class LockboxApplication : Application() {
         injectContext()
         setupLifecycleListener()
 
+        // Glean SDK integration.
+        Glean.setUploadEnabled(SettingStore.shared.sendUsageData.blockingFirst()) // TODO: does this make sense?
+        Glean.initialize(applicationContext, Configuration(channel=BuildConfig.BUILD_TYPE))
+
+        // TODO: subscribe to settings to call `Glean.setUploadEnabled()` on/off?
```

Comment:
How can I detect whether the "can send data" pref value changed? We need to call `Glean.setUploadEnabled(newValue)` in that case.